xbmc/interfaces/legacy/Dialog.cpp

```diff
@@ -85,6 +88,27 @@ namespace XBMCAddon
       return pDialog->IsConfirmed();
     }
 
+    bool Dialog::videoinfo(const ListItem* item)
+    {
+      const AddonClass::Ref<xbmcgui::ListItem> listitem(item);
+      auto fileItem = new CFileItem(*listitem->item);
```

Comment:
Thx! My C++ skills are still WIP, this was just taken from another part of our API code.